src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -120,6 +121,21 @@
      :title           (i18n/label :t/user-gets {:name receiver})
      :subtitle        amount}]])
 
+(defn select-asset-bottom-sheet
```

Comment:
should we move this to wallet/sheets ?? I wonder is this elsewhere in the designs? 🤔

Reply:
I'd say no, but we can move it to another ns if we find out it's reused.